plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -35,7 +35,7 @@
 public final class JdbcTableHandle
         implements ConnectorTableHandle
 {
-    private final JdbcRelationHandle relationHandle;
+    private final JdbcRelation relationHandle;
```

Comment:
I considered changing the field name, but i decided it brings too little benefit to actually do such change and cause downstream costs, of any.
I did not consider changing the class name. `JdbcRelation` seem equally good as `JdbcRelationHandle`.

Reply:
> I considered changing the field name, but i decided it brings too little benefit to actually do such change and cause downstream costs, of any.

I am aware of this and I don't mind to wait for time where it is a bit calm.

> JdbcRelation seem equally good as JdbcRelationHandle.

It is not. "Handle" is something common in Trino and it has a semantic that does not apply here.